src/PKSim.Core/Model/ParameterFactory.cs

```diff
@@ -222,6 +223,8 @@ private void setParameterProperties(IParameter parameter, ParameterMetaData para
          parameter.Info = parameterMetaData.Clone();
          parameter.Dimension = _dimensionRepository.DimensionByName(parameterMetaData.Dimension);
          parameter.ValueOrigin.UpdateFrom(parameterMetaData.ValueOrigin);
+         //Update from does not update the Id of the ValueOrigin. We have to set it by hand based on the meta data values
+         parameter.ValueOrigin.Id = parameterMetaData.ValueOriginId;
```

Comment:
Not so pleased about that. Since UpdateFrom is used to update all the time, it does not update the Id of the ValueOrigin (it needs to be defined one and for all). As such it needs to be set separately. We could define another method in valueOrigin or simply leave it as is